src/realm/object-store/sync/generic_network_transport.hpp

```diff
@@ -205,10 +206,17 @@ struct Request {
     bool uses_refresh_token = false;
 };
 
+enum class ResponseStatus { Success, Failure };
```

Comment:
I would call this `Result` or `ResponseResult`– both because it's not _really_ a status, it's a final result, and to not conflate it with the http status code.

Reply:
Agree, fixed